requirements-dev.txt

```diff
@@ -1,5 +1,5 @@
 #
-# This file is autogenerated by pip-compile with Python 3.10
+# This file is autogenerated by pip-compile with Python 3.11
```

Comment:
Is the Python3.11 necessary here?

Reply:
That's the result of me using Python3.11 for pre-commit. I don't think it really matters because it's just a comment in a automatically generated file.

Reply:
I manually changed it back to 3.10

Reply:
Hmmm, could you actually generated it with Python3.10
It does make a difference actually. The versions generated for the dependencies depends on the Python version, and some modules with specific version may not work for Python3.11